src/core/Akka.MultiNodeTestRunner.TrxReporter/Models/ResultSummary.cs

```diff
@@ -0,0 +1,101 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="ResultSummary.cs" company="Akka.NET Project">
+//      Copyright (C) 2009-2019 Lightbend Inc. <http://www.lightbend.com>
+//      Copyright (C) 2013-2019 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+namespace Akka.MultiNodeTestRunner.AzureDevOps.Models
```

Comment:
Need to adjust the namespace - should be `Akka.MultiNodeTestRunner.TrxReporter`.